decidim-core/app/cells/decidim/redesigned_author_cell.rb

```diff
@@ -119,7 +119,6 @@ def creation_date?
 
     def creation_date
       date_at = from_context.try(:published_at) || from_context.try(:created_at)
-
```

Comment:
Can you please revert the change in this file?

We should avoid doing any changes in files that are not related to the PR.

Reply:
Good point. Noted